migrations/helpers/dart_samples_update_helper.py

```diff
@@ -136,6 +136,8 @@ def migrate_all_dbs(config: Config, s_start_datetime: str = "", s_end_datetime:
             # create connection to the MLWH database
             with create_mysql_connection(config, False) as mlwh_conn:
                 # 5. update the MLWH (should be an idempotent operation)
+
```

Comment:
Use the old SQL_MLWH_MULTIPLE_INSERT instead of the changed one, so this migration can still be applied

Reply:
should we test running the migrations. and if a future filtered positive rule would be ok